storage/gcsupload/README.md

```diff
@@ -0,0 +1,19 @@
+# gcsupload
+
+This is a simple CLI that allows you to upload a file to your Google Cloud Storage bucket.
```

Comment:
link to page on authentication?

https://cloud.google.com/docs/authentication/getting-started